doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+
+For the purposes of this primer you will focus on a simple example, ingesting a static dataset of static entities.  simplemaps.com provides a free list of world cities.  Follow `this link <https://simplemaps.com/data/world-cities>`_ to download this data.  Conduce also provides a version of this list in the ``samples`` directory.
+
+Once you have the data downloaded you must decide how the data should be represented in Conduce.  Each record is described by the following fields:
+
+- city
+- city_ascii
+- lat
+- lng
+- pop
+- country
+- iso2
+- iso3
+- province
+
+These fields are described in detail on the page from which the data is downloaded.  For this example you'll work with all the fields.  However the only fields that are required are ``lat`` and ``lng``.
+
+An entity requires the following fields in order to be ingested: ID, location, and kind.  Kind is a field that describes the type of thing the entity represents.  you'll be ingesting this data as static entities, so a timestamp is not required.  you'll map the source data to Conduce entities as follows::
+
+    {
+        "id": <UUID you will generate>,
+        "kind": "city",
+        "point": {
+            "lat": <lat>,
+            "lon": <lng>
+        },
+        "name": <city>,
+        "long_name": <city_ascii>,
+        "population": <pop>,
+        "province": <province>,
+        "country": <country>,
+        "iso2": <iso2>,
+        "iso3": <iso3>,
+    }
+
+Notice that two of the fields ``id`` and ``kind`` are not derived from the data.  you will generate a unique ID just in case any of the city names are the same.  you will hardcode ``kind`` to city in case you want to ingest other types of data into this dataset in the future.  If you wanted to do something more sophisticated you could categorize the cities by population and set ``kind`` to something like "small_city," "medium_city," and "large_city."
+
+Following this pattern the first city in the dataset takes the following form::
+
+    {
+        "id": str(uuid.uuid4()),
+        "kind": "city",
+        "point": {
+            "lat": 34.9830001,
+            "lon": 63.13329964
+        },
+        "name": "Qal eh-ye",
+        "long_name": "Qal eh-ye Now",
+        "population": 2997,
+        "province": "Badghis",
+        "country": "Afghanistan",
+        "iso2": "AF",
+        "iso3": "AFG",
+    }
+
+Writing the code to iterate over the CSV file and convert each record to a Conduce entity is left to the reader.  However there are utilities included with the Conduce Python API that provide example implementations.
+
+The resulting entities should be compiled into a list.  Once you have generated the entity list it is time to ingest the data to Conduce.
```

Comment:
"ingest the data to Conduce" sounds awkward to me - maybe "into Conduce"?  I can't put my finger on it though

Reply:
Agreed.  I have a hard time with this language.  Send data to Conduce is certainly good.  I suppose the problem is that I am using "Conduce" to refer to the datastore.  I could say "into Conduce's datastore."

Reply:
ingest into sounds weird anyway.  I ingest something.  Someone does not ingest something into me.

So I suppose the correct phrasing would actually be, "send the data to Conduce to be ingested."   Very wordy.

Reply:
That probably means the `ingest_*` API are not named correctly 😿

Reply:
Yea, that may be the case.

Consume is very similar to ingest, and I use that word a lot more.  I think the issue is that Conduce does the consuming, and if sentence has the user or the data doing the ingesting, it sounds off.

Here the user is doing it.  I think the last example sounds good, but agree it is wordy.  It could be, "send the data to Conduce to ingest".